About/FAQ.md

```diff
@@ -161,26 +161,48 @@ Unlike [Tails](https://tails.boum.org/), which aims to be a stateless OS that
 leaves no trace on the computer of its  presence, Heads is intended for the
 case where you need to store data and state on the computer.
 
-HOTP vs TOTP
+HOTP vs TOTP in Heads context
```

Comment:
This section title doesn't hit the mark. TOTP is irrelevant, and HOTP should be "reverse HOTP" if mentioned at all. Perhaps a better section title would be something like "USB security key features used by Heads".

And, add OpenPGP here.